sql/core/src/test/resources/sql-tests/inputs/ansi/decimalArithmeticOperations.sql

```diff
@@ -0,0 +1,4 @@
+-- SPARK-23179: SQL ANSI 2011 states that in case of overflow during arithmetic operations,
+-- an exception should be thrown instead of returning NULL.
+-- This is what most of the SQL DBs do (eg. SQLServer, DB2).
+--import decimalArithmeticOperations.sql
```

Comment:
This seems to fail still.
```
[info] - ansi/decimalArithmeticOperations.sql *** FAILED *** (2 seconds, 158 milliseconds)
```

Reply:
Yea, thanks for your catch-up! I've fixed now.